From 0ec32ea7cc2f9947ede1c87d079f6f34c221cb45 Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Wed, 5 Feb 2014 00:33:39 +0100
Subject: [PATCH] build: stop misplacing documentation

docdir must not be statically set as it would override what the
user gave to ./configure --docdir=...
---
 doc/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 78abbd6..ea73e88 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -1,5 +1,3 @@
-docdir = $(pkgdatadir)/doc
-
 doc_DATA = 4ti2_manual.pdf
 
 ## Manual sources.
-- 
2.19.0

